docs/content/en/docs/topics/repair/_index.md

```diff
@@ -62,7 +71,7 @@ prometheus-k8ssandra-kube-prometheus-stack-prometheus-0     2/2     Running
 
 ## What can I do in Reaper?
 
-To access Reaper, if you are running locally, navigate to [http://repair.localhost:8080/webui/](http://repair.localhost:8080/webui/).
+To access Reaper, if you are running locally, navigate to [http://localhost:8080/webui/](http://localhost:8080/webui/).
```

Comment:
"if you are running locally" is a bit confusing here; I'd move that up to the paragraph on line 48 and explain that they should choose something other than `localhost` if they're _not_ running locally, then add a note here to use whatever hostname you specified for `reaper.ingress.host`.

Reply:
Updated text with notes around lines 48 and 74, thanks Jake.